messaginginapp/src/test/java/io/customer/messaginginapp/ui/BasicIntegrationTest.kt

```diff
@@ -0,0 +1,116 @@
+package io.customer.messaginginapp.ui
+
+import io.customer.commontest.config.testConfigurationDefault
+import io.customer.commontest.extensions.attachToSDKComponent
+import io.customer.commontest.extensions.flushCoroutines
+import io.customer.commontest.extensions.random
+import io.customer.commontest.util.ScopeProviderStub
+import io.customer.messaginginapp.MessagingInAppModuleConfig
+import io.customer.messaginginapp.ModuleMessagingInApp
+import io.customer.messaginginapp.di.inAppMessagingManager
+import io.customer.messaginginapp.gist.GistEnvironment
+import io.customer.messaginginapp.gist.data.listeners.GistQueue
+import io.customer.messaginginapp.gist.presentation.GistProvider
+import io.customer.messaginginapp.gist.presentation.GistSdk
+import io.customer.messaginginapp.state.InAppMessagingAction
+import io.customer.messaginginapp.state.InAppMessagingManager
+import io.customer.messaginginapp.testutils.core.JUnitTest
+import io.customer.sdk.core.di.SDKComponent
+import io.customer.sdk.core.util.Logger
+import io.customer.sdk.core.util.ScopeProvider
+import io.customer.sdk.data.model.Region
+import io.mockk.mockk
+import io.mockk.spyk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+/**
+ * Basic integration tests that verify essential functionality without complex mocking.
+ */
+class BasicIntegrationTest : JUnitTest() {
```

Comment:
Can we rename `Basic` prefix to something more descriptive e.g. `MessagingInAppIntegrationTest` etc. As `Basic` feels like some base class at first look 😅

Reply:
updated